bundles/config/org.eclipse.smarthome.config.discovery/OSGI-INF/InboxAutoIgnore.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I was not sure if I would then need to convert all other component descriptors as well. Would I?